akka-testkit-typed/src/main/scala/akka/testkit/typed/BehaviorTestkit.scala

```diff
@@ -178,8 +172,17 @@ class BehaviorTestkit[T] private (_name: String, _initialBehavior: Behavior[T])
   }
 
   private var current = Behavior.validateAsInitial(Behavior.undefer(_initialBehavior, ctx))
+  private var currentUncanonical = current
 
+  /**
+   * Returns the current canonical behavior
+   */
   def currentBehavior: Behavior[T] = current
+
+  /**
+   * Returns the current behavior before being converted to the canonical one
+   */
+  def currentUncanonicalBehavior: Behavior[T] = currentUncanonical
```

Comment:
I don't like the "word" uncanonical... If anyone has a better alternative :)

Reply:
`returnedBehavior` ?